docs/data/material/customization/overriding-component-structure/overriding-component-structure.md

```diff
@@ -0,0 +1,103 @@
+# Overriding component structure
+
+<p class="description">Learn how to override the default DOM structure of Material UI components.</p>
+
+Material UI components are designed to suit the widest possible range of use cases, but you may occasionally need to change how a component's structure is rendered in the DOM.
+
+To understand how to do this, it helps to know a bit about how the API design has evolved over time, and to have an accurate mental model of the components themselves.
+
+## Context
+
+Prior to Material UI v6, it was not possible to override the structure of most components in the library.
+Some components had `*Props` props that allowed you to pass props to a specific slot, but this pattern was not applied consistently.
+
+In v6, those props were deprecated in favor of the `slots` and `slotProps` props, which allow for more granular control over the structure of a component and make the API more consistent across the library.
+
+To see the available slots for a component, refer to the slots sections of the respective component API documentation.
+
+## The mental model
+
+A component's structure is determined by the elements that fill that component's **slots**.
+Slots are most commonly filled by HTML tags, but may also be filled by React components.
+
+All components contain a root slot that defines their primary node in the DOM tree; more complex components also contain additional interior slots named after the elements they represent.
```

Comment:
```suggestion
All components contain a root slot that defines their primary node in the DOM tree; more complex components also contain additional interior slots named after the elements they represent.
To see the available slots for a component, refer to the slots sections of the respective component API documentation.
```

Reply:
could also consider putting this in a callout - I don't have a strong opinion either way.

```suggestion
All components contain a root slot that defines their primary node in the DOM tree; more complex components also contain additional interior slots named after the elements they represent.

:::info
To see the available slots for a component, refer to the slots sections of the respective component API documentation.
:::
```